src/test/java/utils/GeneralConfigModule.java

```diff
@@ -21,15 +21,28 @@ public void applyToConfigBuilder(SchemaGeneratorConfigBuilder builder) {
 		builder.forTypesInGeneral()
 				.withSubtypeResolver(new ClassGraphSubtypeResolver())
 				.withCustomDefinitionProvider(descriptionProvider)
-				.withTypeAttributeOverride(descriptionProvider);
+				.withTypeAttributeOverride(descriptionProvider)
+				//.withCustomDefinitionProvider(new ObjectIdToCustomSchemaProvider())
```

Comment:
The `CustomDefinitionProvider` gives you full control over the generated schema, but you have to manually include all relevant aspects.